checkov/common/runners/runner_registry.py

```diff
@@ -53,9 +53,9 @@ def print_reports(self, scan_reports, args, url=None):
                     junit_reports.append(report)
                     # report.print_junit_xml()
                 elif args.output == 'github_failed_only':
-                    report.print_failed_github_md()
+                    report.print_failed_github_md(use_bc_ids=args.use_bc_ids)
                 else:
-                    report.print_console(is_quiet=args.quiet, is_compact=args.compact)
+                    report.print_console(is_quiet=args.quiet, is_compact=args.compact, use_bc_ids=args.use_bc_ids)
```

Comment:
this method is also used upstream. let's use a setter.

Reply:
The only other usage is in tests.

Reply:
I think we should refactor the reports output into different classes, so that the Report class is just a data object. And probably runner_registry should not be having all the logic for determining how to print reports, so we can refactor that too.

I prefer to keep this for now and handle the refactor separately. I'm 99% sure that there is not another entrypoint to `print_reports` (it just takes `args` anyways) or `print_console`.